packages/workbox-core/utils/assert.mjs

```diff
@@ -27,7 +27,37 @@ const isArray = (value, {moduleName, className, funcName, paramName}) => {
   }
 };
 
+const hasMethod = (object, expectedMethod,
+                   {moduleName, funcName, paramName}) => {
+  const type = typeof object[expectedMethod];
+  if (type !== 'function') {
+    throw new WorkboxError('not-a-method', {paramName, expectedMethod,
```

Comment:
Nit: could rename the error key to 'missing-a-method'.

Reply:
Done.